linkis-spring-cloud-services/linkis-service-gateway/linkis-spring-cloud-gateway/src/main/java/org/apache/linkis/gateway/springcloud/http/GatewayAuthorizationFilter.java

```diff
@@ -159,10 +159,14 @@ private Mono<Void> gatewayDeal(
     ServiceInstance serviceInstance;
     try {
       parser.parse(gatewayContext);
+      logger.info("client request uri: " + gatewayContext.getRequest().getRequestURI());
```

Comment:
If you want to know the requestURI, I think it would be better to add it at the beginning of GatewayAuthorizationFilter.
Whether the level can be changed to debug？